Screenbox/App.xaml

```diff
@@ -88,6 +88,8 @@
                     <Thickness x:Key="PageThumbnailMargin">52,0</Thickness>
                     <Thickness x:Key="PageThumbnailMarginMinimal">14,0</Thickness>
 
+                    <Thickness x:Key="TopMediumMargin">0,12,0,0</Thickness>
+                    <Thickness x:Key="TopLargeMargin">0,16,0,0</Thickness>
```

Comment:
I'm a bit confused. When do we use Medium or Large margin? Is there a pattern for when to use each one?

Reply:
Not really and we should. I'll have to backtrack to see why but it seems to me that I started with those two values but then after testing touch targets the Large key fitted better. Maybe the Medium key stuck because of the BreadcrumbBar, at least on the VideosPage.

Reply:
Yeah, can confirm that the VideosPage Medium margin is there to match [this](https://github.com/huynhsontung/Screenbox/blob/main/Screenbox/Pages/FolderViewPage.xaml#L73).

The other usecase is more subjective, the "center" between the 'Shuffle & Play' button and the music content.